applications/GeoMechanicsApplication/custom_utilities/process_utilities.cpp

```diff
@@ -45,4 +71,31 @@ std::vector<std::reference_wrapper<ModelPart>> ProcessUtilities::GetModelPartsFr
     return result;
 }
 
+void ProcessUtilities::AddProcessesSubModelPartListToSolverSettings(const Parameters& rProjectParameters,
+                                                                    Parameters& rSolverSettings)
+{
+    std::set<std::string, std::less<>> domain_condition_names;
```

Comment:
Also here I expect we can omit std::less<>, since it seems to be the default (see https://en.cppreference.com/w/cpp/container/set.html)
```suggestion
    std::set<std::string> domain_condition_names;
```

Reply:
it is added to remove a code smell